src/App.tsx

```diff
@@ -1,14 +1,38 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { useAppSelector, useAppDispatch } from './app/hooks';
+import { actions as Featuresactions } from './features/todos';
```

Comment:
```suggestion
import { actions as todosActions } from './features/todos';
```
Why would you name it Featuresactions? It's so incorrect in many ways:
1. Don't name regular variables with the first capital letter.
2. Use camelCase: `typeWordsLikeThis`.
3. Did you watch videos? Misha has named actions based on the part of the state they represent. So in this case it'd be `todosActions`.